portfolio/src/main/java/com/google/sps/data/MapMarker.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MapMarker {
```

Comment:
@ccondit 
No action needed. Just FYI. Internally we use AutoValue library to generate such classes automatically. [It looks like it's open source](https://github.com/google/auto/blob/master/value/userguide/index.md#how-to-use-autovalue) so we could use it for our capstone project. That being said protobuf would have made it even easier for this MapMarker use-case as it wold have take care of wire transportation as well.